ui/apps/dev-server-ui/src/app/(dashboard)/stream/StreamDetails.tsx

```diff
@@ -89,6 +89,11 @@ export default function StreamDetails() {
     );
   };
 
+  let events = [eventResult.data];
+  if (runResult?.data?.run?.events && runResult.data.run.events.length > 1) {
```

Comment:
Will the `events` resolver include a single event when it isn't a batch? Like what Cloud does

Reply:
yes it will